templates/nbde_client.conf

```diff
@@ -0,0 +1 @@
+hostonly_cmdline=yes
```

Comment:
Looking at this again, it may make sense to have `kernel_cmdline="rd.neednet=1"` here instead of `hostonly_cmdline=yes`, to have network enabled in the initrd unconditionally.

@Freakygnome: are you planning on updating this PR and marking it as ready to review?

Reply:
Yes would details on the template need to be added to the readme?

Reply:
> Yes would details on the template need to be added to the readme?

No - the README is only for documenting the public api, not implementation details.